### ISSUE TITLE
How can i trust this?

### ISSUE THREAD

**Issue by watcha27** [0xVienna/Celery#5]
its existence only started only yesterday, do yall have a discord?

**Comment by 0xVienna** [comment]
Yes my discord tag is …WinVienna!!#8724 also this safe is a mirror but new updates may take up to 2 days. Also this is fanmade website

**Comment by Yxi1d** [comment]
> 

He's saying "do you have a discord server for celery", he isn't asking for your Discord Username
